generators/add-franklin/index.js

```diff
@@ -0,0 +1,87 @@
+const Generator = require('yeoman-generator')
+const path = require('path')
+const { addDependencies, addPkgScript, writeKeyAppConfig } = require('../../lib/utils')
+const { isLoopingPrompts } = require('../../lib/constants')
+
+class FranklinGenerator extends Generator {
+  constructor (args, opts) {
+    super(args, opts, undefined)
+    this.props = {}
+  }
+
+  async prompting () {
+    // Default tenant if we skip all prompts
+    this.tenant = 'demo-site'
+    this.coreComponents = []
+    if (!this.options['skip-prompt']) {
+      const res = await this.prompt([
+        {
+          type: 'input',
+          name: 'tenant',
+          message: 'What is the name of the AEM tenant you would like to use?',
+          loop: isLoopingPrompts
+        },
+        {
+          type: 'checkbox',
+          name: 'coreComponents',
+          message: 'Which AEM Core Components should we generate stubs to customise for?',
```

Comment:
Not sure if I would understand this prompt. Maybe "Which AEM Core Components are you planning to customize? We'll generate stubs for those".